tensorflow/python/keras/distribute/saved_model_save_load_test.py

```diff
@@ -32,6 +33,8 @@
 from tensorflow.python.saved_model import saved_model
 
 
+@testing_utils.run_all_without_tensor_float_32(
+    'Uses Dense layers, which call matmul')
```

Comment:
We should adjust the tolerance for this test as well instead of disabling it for TF32.

Reply:
See my previous comment